journalbeat/magefile.go

```diff
@@ -22,20 +22,47 @@ package main
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/dev-tools/mage"
 )
 
 func init() {
 	mage.BeatDescription = "Journalbeat ships systemd journal entries to Elasticsearch or Logstash."
 
-	// TODO filter platforms
+	mage.Platforms = mage.Platforms.Filter("linux !linux/ppc64 !linux/mips64")
 }
 
+var (
+	deps = map[string]func() error{
+		"linux/386":      installLinux386,
+		"linux/amd64":    installDefaultLinux,
+		"linux/arm64":    installLinuxArm64,
+		"linux/armv5":    installLinuxArmLe,
+		"linux/armv6":    installLinuxArmLe,
+		"linux/armv7":    installLinuxArmHf,
+		"linux/mips":     installLinuxMips,
+		"linux/mipsle":   installLinuxMipsle,
+		"linux/mips64le": installLinuxMips64le,
+		"linux/ppc64le":  installLinuxPpc64le,
+		"linux/s390x":    installLinuxs390x,
+
+		// No deb packages
+		//"linux/ppc64": installLinuxPpc64,
+		//"linux/mips64": installLinuxMips64,
+	}
+)
+
+const (
+	currentLibsystemdDev = "libsystemd-dev"
```

Comment:
"current" and "prev" don't convey much information to me. Do names like `libsystemdDevPkgNameCurrent` and `libsystemdDevPkgNameDebian8` match up with their expected usage?

Reply:
I removed the second `const` because on Debian 8 `libsystemd-dev` is already available.